Build appears to be broken: error: ISO C++ forbids declaration of ‘DateNavigationWidget’ with no type, etc.

Looks like it is broken:
```
--->  Configuring notes-qt4
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/NotesManager-0.0.3_494b3dc5bf0e827612dfdddc83c11073b07d10d8" && /opt/local/libexec/qt4/bin/qmake PREFIX=/opt/local QMAKE_CC=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cc/usr/bin/gcc-4.2 QMAKE_CXX=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cxx/usr/bin/g++-4.2 QMAKE_OBJC=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/objc/usr/bin/gcc-4.2 QMAKE_CFLAGS="-I/opt/local/include -Os -arch ppc" QMAKE_CXXFLAGS="-I/opt/local/include -Os -arch ppc" QMAKE_LFLAGS="-I/opt/local/include -L/opt/local/lib -Wl,-headerpad_max_install_names" QMAKE_LINK_C=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cc/usr/bin/gcc-4.2 QMAKE_LINK_C_SHLIB=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cc/usr/bin/gcc-4.2 QMAKE_LINK=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cxx/usr/bin/g++-4.2 QMAKE_LINK_SHLIB=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cxx/usr/bin/g++-4.2 
Project MESSAGE: qtAddLibrary: found framework QtWebKit in directory /opt/local/libexec/qt4/Library/Frameworks
Project MESSAGE: qtAddLibrary: found framework QtScript in directory /opt/local/libexec/qt4/Library/Frameworks
Project MESSAGE: qtAddLibrary: found framework QtScriptTools in directory /opt/local/libexec/qt4/Library/Frameworks
Project MESSAGE: qtAddLibrary: found framework QtSql in directory /opt/local/libexec/qt4/Library/Frameworks
Project MESSAGE: qtAddLibrary: found framework QtXml in directory /opt/local/libexec/qt4/Library/Frameworks
Project MESSAGE: qtAddLibrary: found framework QtGui in directory /opt/local/libexec/qt4/Library/Frameworks
Project MESSAGE: qtAddLibrary: found framework QtCore in directory /opt/local/libexec/qt4/Library/Frameworks
--->  Building notes-qt4
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/NotesManager-0.0.3_494b3dc5bf0e827612dfdddc83c11073b07d10d8" && /usr/bin/make -j1 -w all 
make: Entering directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/NotesManager-0.0.3_494b3dc5bf0e827612dfdddc83c11073b07d10d8'
/opt/local/libexec/qt4/bin/uic Manager/Manager.ui -o ui_Manager.h
/opt/local/libexec/qt4/bin/uic Manager/pages/page_settings.ui -o ui_page_settings.h
/opt/local/libexec/qt4/bin/uic Manager/pages/page_notes.ui -o ui_page_notes.h
/opt/local/libexec/qt4/bin/uic Manager/pages/page_about.ui -o ui_page_about.h
/opt/local/libexec/qt4/bin/uic Note/AttachPanel.ui -o ui_AttachPanel.h
/opt/local/libexec/qt4/bin/uic Note/settingsnotedialog.ui -o ui_settingsnotedialog.h
/opt/local/libexec/qt4/bin/uic FindAndReplace/findandreplace.ui -o ui_findandreplace.h
FindAndReplace/findandreplace.ui: Warning: Z-order assignment: '' is not a valid widget.
FindAndReplace/findandreplace.ui: Warning: Z-order assignment: 'bottomVerticalSpacer' is not a valid widget.
/opt/local/libexec/qt4/bin/uic FormattingToolbar/DialogInsertHyperlink.ui -o ui_DialogInsertHyperlink.h
/opt/local/libexec/qt4/bin/uic FormattingToolbar/FormattingToolbar.ui -o ui_FormattingToolbar.h
/opt/local/libexec/qt4/bin/uic FormattingToolbar/dialoginserthorizontalline.ui -o ui_dialoginserthorizontalline.h
/opt/local/libexec/qt4/bin/uic NavigationPanel/noteeditwidget.ui -o ui_noteeditwidget.h
/opt/local/libexec/qt4/bin/uic NavigationPanel/foldernavigationwidget.ui -o ui_foldernavigationwidget.h
/opt/local/libexec/qt4/bin/uic NavigationPanel/tagsnavigationwidget.ui -o ui_tagsnavigationwidget.h
/opt/local/libexec/qt4/bin/uic NavigationPanel/datenavigationwidget.ui -o ui_datenavigationwidget.h
/opt/local/libexec/qt4/bin/uic ScriptModule/scriptsmanager.ui -o ui_scriptsmanager.h
/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/compwrap/cxx/usr/bin/g++-4.2 -c -I/opt/local/include -Os -arch ppc -O2 -arch ppc -Wall -W -DQT_NO_DEBUG -DQT_WEBKIT_LIB -DQT_SCRIPT_LIB -DQT_SCRIPTTOOLS_LIB -DQT_SQL_LIB -DQT_XML_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_SHARED -I/opt/local/libexec/qt4/share/mkspecs/macx-g++ -I. -I. -I/opt/local/libexec/qt4/Library/Frameworks/QtCore.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtGui.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtXml.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtSql.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtScriptTools.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtScript.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtWebKit.framework/Versions/4/Headers -IManager -IFancyTabBar -IImageCropper -INote -IFindAndReplace -IFadLineEdit -IFormattingToolbar -INavigationPanel -IComponents/qt-solutions/qtsingleapplication/src -IComponents/quazip/quazip-0.6.1/src -IComponents/QScintilla-gpl-2.8.1/Qt4Qt5 -I/opt/local/libexec/qt4/Library/Frameworks/QtWebKit.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtScript.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtScriptTools.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtSql.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtXml.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtGui.framework/Versions/4/Headers -I/opt/local/libexec/qt4/Library/Frameworks/QtCore.framework/Versions/4/Headers -I/opt/local/libexec/qt4/include -I. -F/opt/local/libexec/qt4/Library/Frameworks -F/opt/local/libexec/qt4/lib -o Manager.o Manager/Manager.cpp
In file included from Manager/pages/page_notes.h:12,
                 from Manager/Manager.h:12,
                 from Manager/Manager.cpp:1:
./ui_page_notes.h:26:50: error: NavigationPanel\datenavigationwidget.h: No such file or directory
In file included from Manager/pages/page_notes.h:12,
                 from Manager/Manager.h:12,
                 from Manager/Manager.cpp:1:
./ui_page_notes.h:55: error: ISO C++ forbids declaration of ‘DateNavigationWidget’ with no type
./ui_page_notes.h:55: error: expected ‘;’ before ‘*’ token
./ui_page_notes.h: In member function ‘void Ui_Page_notes::setupUi(QMainWindow*)’:
./ui_page_notes.h:166: error: ‘tab_Dates’ was not declared in this scope
./ui_page_notes.h:166: error: expected type-specifier before ‘DateNavigationWidget’
./ui_page_notes.h:166: error: expected `;' before ‘DateNavigationWidget’
./ui_page_notes.h: In member function ‘void Ui_Page_notes::retranslateUi(QMainWindow*)’:
./ui_page_notes.h:338: error: ‘tab_Dates’ was not declared in this scope
make: *** [Manager.o] Error 1
make: Leaving directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/NotesManager-0.0.3_494b3dc5bf0e827612dfdddc83c11073b07d10d8'
Command failed:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_sysutils_notes-qt4/notes-qt4/work/NotesManager-0.0.3_494b3dc5bf0e827612dfdddc83c11073b07d10d8" && /usr/bin/make -j1 -w all 
Exit code: 2
```

--- a/Manager/pages/page_notes.ui
+++ b/Manager/pages/page_notes.ui
@@ -392,7 +392,7 @@
   <customwidget>
    <class>DateNavigationWidget</class>
    <extends>QWidget</extends>
-   <header>NavigationPanel\datenavigationwidget.h</header>
+   <header>NavigationPanel/datenavigationwidget.h</header>
    <container>1</container>
   </customwidget>
   <customwidget>

